model_evaluation/input_fn_example.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Defines some examples of input_fn for the evaluation notebook."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import numpy as np
+import pandas as pd
+import pkg_resources
+import os
+import random
+import re
+
+import tensorflow as tf
+from tensorflow.python.lib.io import file_io
+
+from unintended_ml_bias import model_bias_analysis
+from utils_export import utils_tfrecords
+
+
+#Faster to access GCS file + https://github.com/tensorflow/tensorflow/issues/15530
+os.environ['GCS_READ_CACHE_MAX_SIZE_MB'] = '0'
+
+
+#TODO(fprost): Clean this file.
+
+#### #### #### #### #### ####
+#### PERFORMANCE DATASET ####
+#### #### #### #### #### ####
+
+
+def create_input_fn_toxicity_performance(tokenizer, model_input_comment_field):
+  """Generates an input_fn to evaluate model performance on toxicity dataset."""
+
+  TOXICITY_PERFORMANCE_DATASET = 'gs://kaggle-model-experiments/resources/toxicity_q42017_test.tfrecord'
```

Comment:
Update this to use the new storage bucket

Reply:
As well as all the other paths

Reply:
Done. Also for the ones in notebook